CHANGELOG.md

```diff
@@ -1,228 +1,178 @@
-[13]: https://github.com/magento-commerce/venia-sample-data-modules/pull/13
-[3538]: https://github.com/magento/pwa-studio/pull/3538
-[3529]: https://github.com/magento/pwa-studio/pull/3529
-[11]: https://github.com/magento-commerce/magento2-upward-connector/pull/11
-[3559]: https://github.com/magento/pwa-studio/pull/3559
-[39]: https://github.com/magento-commerce/pwa-tests/pull/39
-[3564]: https://github.com/magento/pwa-studio/pull/3564
-[3555]: https://github.com/magento/pwa-studio/pull/3555
-[3552]: https://github.com/magento/pwa-studio/pull/3552
-[101]: https://github.com/magento-commerce/pwa-studio-cicd/pull/101
-[3561]: https://github.com/magento/pwa-studio/pull/3561
-[15]: https://github.com/magento-commerce/magento2-pwa/pull/15
-[3654]: https://github.com/magento/pwa-studio/pull/3654
-[3624]: https://github.com/magento/pwa-studio/pull/3624
-[14]: https://github.com/magento-commerce/magento2-pwa/pull/14
-[3619]: https://github.com/magento/pwa-studio/pull/3619
-[18]: https://github.com/magento-commerce/magento2-pwa/pull/18
-[3542]: https://github.com/magento/pwa-studio/pull/3542
-[3551]: https://github.com/magento/pwa-studio/pull/3551
-[3567]: https://github.com/magento/pwa-studio/pull/3567
-[3571]: https://github.com/magento/pwa-studio/pull/3571
-[3520]: https://github.com/magento/pwa-studio/pull/3520
-[3515]: https://github.com/magento/pwa-studio/pull/3515
-[3524]: https://github.com/magento/pwa-studio/pull/3524
-[3553]: https://github.com/magento/pwa-studio/pull/3553
-[3575]: https://github.com/magento/pwa-studio/pull/3575
-[16]: https://github.com/magento-commerce/magento2-upward-connector/pull/16
-[6]: https://github.com/magento-commerce/upward-php/pull/6
-[3599]: https://github.com/magento/pwa-studio/pull/3599
-[3622]: https://github.com/magento/pwa-studio/pull/3622
-[3608]: https://github.com/magento/pwa-studio/pull/3608
-[3573]: https://github.com/magento/pwa-studio/pull/3573
-[3603]: https://github.com/magento/pwa-studio/pull/3603
-[3584]: https://github.com/magento/pwa-studio/pull/3584
-[3625]: https://github.com/magento/pwa-studio/pull/3625
-[3655]: https://github.com/magento/pwa-studio/pull/3655
-[18]: https://github.com/magento-commerce/magento2-upward-connector/pull/18
-[16]: https://github.com/magento-commerce/venia-sample-data-modules/pull/16
-[3658]: https://github.com/magento/pwa-studio/pull/3658/files
-[3664]: https://github.com/magento/pwa-studio/pull/3664
-[3665]: https://github.com/magento/pwa-studio/pull/3665
+| `venia-concept`                     | **12.3.0**     |
+| `venia-ui`                          | **9.3.0**      |
+| `magento2-pwa`                      | **0.2.0**      |
+| `magento2-pwa-commerce`             | **0.0.1**      |
+| `magento-venia-sample-data-modules` | **0.0.2**      |
+| `magento-venia-sample-data-modules-ee`| **0.0.1**    |
+| `magento2-upward-connector`         | **2.0.1**      |
+| `upward-php`                        | **2.0.1**      |
```

Comment:
@dobooth I updated test plan. no change for upward repo's.
magento2-upward-connector >> 2.0.0 
upward-php >> 2.0.0